deploy.sh

```diff
@@ -82,9 +82,15 @@ configure_compilers() {
 
 populate_mirror() {
     what=$1
+    log "populating mirror for ${what}"
+
     specfile="$(install_dir ${what})/data/specs.txt"
+    spec_list=$(spack filter --not-installed $(cat ${specfile}))
```

Comment:
As I mentioned before, maybe we should save this in a separate file? It would then be very easy to see what was attempted to be installed.